dll/directx/wine/ddraw/ddraw.c

```diff
@@ -1145,6 +1145,11 @@ static HRESULT WINAPI ddraw7_SetDisplayMode(IDirectDraw7 *iface, DWORD width, DW
                 ddrawformat_from_wined3dformat(&ddraw->primary->surface_desc.u4.ddpfPixelFormat, mode.format_id);
         }
         ddraw->flags |= DDRAW_RESTORE_MODE;
+#ifdef __REACTOS__
+
+		if (ddraw->cooperative_level & DDSCL_EXCLUSIVE)
```

Comment:
The indent looks strange

Reply:
yes, tabs. yuck. Since it is within an ifdef _REACTOS block, we could space-indent like the surrounding code is.

Reply:
done, original code where i copied from used tabs